schemas/package.yml

```diff
@@ -0,0 +1,96 @@
+---
+- name: package
+  title: Package
+  group: 2
+  short: These fields contain information about an installed software package.
+  description: >
+    These fields contain information about an installed software package. It contains general information about a
+    package, such as name, version or size. It also contains installation details, such as time, URL or
+    user who installed the package.
+  type: group
+  fields:
+
+    - name: name
+      level: core
+      type: keyword
+      description: Package name
+      example: go
+
+    - name: version
+      level: core
+      type: keyword
+      description: Package version
+      example: 1.12.9
+
+    - name: description
+      level: core
+      type: keyword
+      description: Description of the package.
+      example: Open source programming language to build simple/reliable/efficient software.
+
+    - name: size
+      format: string
+      level: core
+      type: long
+      description: Package size in bytes.
+      example: 62231
+
+    - name: installed
+      level: core
+      type: date
+      description: Time when package was installed.
+
+    - name: type
+      level: core
+      type: keyword
+      description: Information about package type. When installed from a package manager, this would contain the
+        package manager name.
+      example: npm
+
+    - name: url.original
+      level: core
+      type: keyword
+      description: URL from where the package was installed.
+      example: https://docs.docker.com/compose/
+
+    - name: path
+      level: core
+      type: keyword
+      description: Path where the package is installed.
+      example: /usr/local/Cellar/go/1.12.9/
+
+    - name: remote_repository
+      level: extended
+      type: keyword
+      description: Remote repository from where the package was installed.
+      example: https://artifacts.elastic.co/packages/7.x/yum
+
+    - name: architecture
+      level: extended
+      type: keyword
+      description: Package architecture.
+      example: runtime
+
+    - name: detailed_version
+      level: extended
+      type: keyword
+      description: Detailed version of the installed package, for instance git commit when using an unreleased version.
+      example: 9b52cc0d83f794f6fc6190f603fa2f62dd9aa2017165d19ff1d62742955a7a05
+
+    - name: checksum
+      level: extended
+      type: keyword
+      description: Checksum of the installed package for verification.
+      example: 9b52cc0d83f794f6fc6190f603fa2f62dd9aa2017165d19ff1d62742955a7a05
+
+    - name: install_scope
+      level: extended
+      type: keyword
+      description: Indicating how the package was installed, e.g. user-local, global.
+      example: global
+
+    - name: license
```

Comment:
I like this.

I would add a suggestion in the definition, to try to use the license identifier from https://spdx.org/licenses/, if possible (and if it's open source). Not a hard requirement, though. Plus there's no listing that I know of for popular non-OS licenses anyway. Just meant as a nudge to try to keep this clean. We want the short name, if possible.